packages/app/src/server/models/config.ts

```diff
@@ -135,6 +135,8 @@ export const defaultCrowiConfigs: { [key: string]: any } = {
   'customize:isEnabledStaleNotification': false,
   'customize:isAllReplyShown': false,
   'customize:isSearchScopeChildrenAsDefault': false,
+  'customize:isDefaultLogo': true,
+  'customize:uploadedLogoSrc': undefined,
```

Comment:
config.ts を拡張する必要はない。Attachment に BRAND_LOGO タイプのドキュメントがあればカスタムロゴ、なければデフォルトロゴなんだから。

Reply:
You don't need to add new data to this file because If  `attachmentType` of a document is BRAND_LOGO, then it is a customized logo. Otherwise it is a Default logo.

Reply:
- Removed definition of `customize:isDefaultLogo` and `customize:uploadedLogoSrc`